src/test/java/PasswordGeneratorTest.java

```diff
@@ -62,7 +63,7 @@ public void checkSecretPassword() {
                 break;
             }
         }
-
+        //this assert is a bit redundant
          assertThat(bCryptPasswordEncoder.matches(actualPassword,
```

Comment:
Granted it's a bit of a weird test but not sure this assert is actually adding anything as we've already checked with a match - maybe we could just print out the answer?